app/src/main/java/com/example/scoutconcordia/CalendarActivity.java

```diff
@@ -0,0 +1,226 @@
+        signIn();
+    }
+
+    public void continueActivity(){
+        showCalendarsList(calendarService);
+
+        //use selectedCalendar as the id to fetch events
+    }
+
+    public void displayCalendar(String id){
+        try {
+            CalendarListEntry calendar = calendarService.calendarList().get(id).execute();
+            Toast.makeText(this, calendar.getSummary(), Toast.LENGTH_LONG).show();
+
+        }catch (Throwable t){}
+    }
+
+    public Dialog onCreateDialogSingleChoice(ArrayList<String> calendars, final HashMap<String, String> calendarIds) {
+
+        final CharSequence[] cs = calendars.toArray(new CharSequence[calendars.size()]);
+        AlertDialog.Builder builder = new AlertDialog.Builder(this, R.style.MaterialThemeDialog);
+
+        builder.setTitle("Select schedule calendar")
+                .setCancelable(false)
+                .setSingleChoiceItems(cs, 1, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        selectedCalendar = cs[which].toString();
+                    }
+                })
+                .setPositiveButton("Select", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int id) {
+
+                        displayCalendar(calendarIds.get(selectedCalendar));
+                    }
+                })
+                .setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int id) {
+                        Intent mapIntent = new Intent(CalendarActivity.this, MapsActivity.class);
+                        startActivity(mapIntent);
+                        CalendarActivity.this.overridePendingTransition(0, 0);
+                    }
+                });
+
+        return builder.create();
+    }
+
+    public void showCalendarsList(Calendar service){
+        HashMap<String, String> calendarIds = new HashMap<String, String>();
+        ArrayList<String> calendars =  new ArrayList<>();
+        try {
+            String pageToken = null;
+            do {
+                CalendarList calendarList = service.calendarList().list().setPageToken(pageToken).execute();
+                List<CalendarListEntry> items = calendarList.getItems();
+
+                for (CalendarListEntry calendarListEntry : items) {
+                    //Log.d("CALENDARSSS: ",calendarListEntry.getSummary());
+                    calendars.add(calendarListEntry.getSummary());
+                    calendarIds.put(calendarListEntry.getSummary(), calendarListEntry.getId());
+                }
+                pageToken = calendarList.getNextPageToken();
+            } while (pageToken != null);
+
+//            return
+        }
+        catch(Throwable t){
+        }
```

Comment:
There should be some kind of action or error message at this point. Also, is there a reason you're catching a Throwable instead of Exception?